### ISSUE TITLE
Clarification on which shells/OS are supported

### ISSUE THREAD

**Issue by kieranjol** [dd388/crals#3]
Hi!

Your site is fantastic and I look forward to seeing it grow. Are ye sticking to bash commands/scripts exclusively? I haven't noticed any Windows commands, but at the same time, I didn't see any bash-specific language in the documentation. Have ye any desire to support windows-only commands, and if so , how should they be differentiated from the bash?

**Comment by dd388** [comment]
Good point! We have been focusing on bash commands only, but that's not totally clear in the writeup. I can make it more explicit. I know I've been sticking to bash because it's what I know best and I can test the commands to make sure they work before I add them in. I hesitate to add Windows commands because I personally couldn't verify them, but perhaps someone else can!

That said, I think it would be super helpful to have Windows commands in there as well -- though I would need help framing them so they have the appropriate context. 

What is the best way to do this for now? I can create a Windows-only section that has all Windows scripts in it. That seems like the most straightforward way to go about it to start, and then we can reconfigure if necessary. What do you think? Do you have any Windows-only commands and explanations to contribute? Also, if I post commands in this thread, would you be able to provide explanations for them? (I have a few Windows shell commands generously forwarded to me by a co-worker.)

**Comment by kieranjol** [comment]
I could verify Windows commands if it came to it. Our office computers are Windows, while our media stations are OSX or Ubuntu. 
I think the windows-only section sounds great! 
I don't have a lot of windows commands, as most of my scripts use python as I can't rely on any one terminal due to our variety of operating systems. I have found a need to use something like `robocopy` (built in windows version of `cp` - https://github.com/kieranjol/IFIscripts/blob/master/move.py#L95), as it seems to be the best way to copy files while maintaining attributes in windows. I occasionally do things like export directory trees to a text file, and probably more but I can't think off the top of my head. The actual windows `cmd` terminal is painfully limited compared to bash.
I would totally provide explanations/test the windows commands from your co-workers.
